docs/design/dab-validate.md

```diff
@@ -0,0 +1,53 @@
+# Design Specification: Validating a Config File
+
+## Overview
+
+This design specification outlines the process for validating the Data API builder config file used by a runtime engine. The validation process ensures that the config file is correctly formatted and contains all the required information for the runtime engine to function correctly.
```

Comment:
what do you mean by "correctly formatted"?
- compliant with the schema OR/(and)
- correct usage of spaces/tabs/indentation?

Reply:
compliant with schema. will update to reflect it correctly.